src/pages/Vaults/Vault/UpdateCollateralModal/index.tsx

```diff
@@ -131,6 +132,8 @@ const UpdateCollateralModal = ({
   );
   useErrorHandler(vaultCollateralizationError);
 
+  const handleClose = chain(() => resetField(COLLATERAL_TOKEN_AMOUNT), onClose);
```

Comment:
added this because I had errors from deposit modal being dragged to the withdraw modal when openned